Source/SpatialGDK/Private/SpatialNetDriver.cpp

```diff
@@ -169,6 +169,44 @@ bool USpatialNetDriver::IsLevelInitializedForActor(const AActor* InActor, const
 	return true;
 }
 
+
+void USpatialNetDriver::NotifyActorDestroyed(AActor* ThisActor, bool IsSeamlessTravel /*= false*/)
```

Comment:
Maybe a comment about how this override is copied from native Unreal, and no longer relevant functionality removed?

Reply:
Yes, will combine with Giray's comment about intentionally not calling Super, if that's okay, and move the explanation in the middle ("The native Unreal driver...") up to the beginning.